lark-stubs/load_grammar.pyi

```diff
@@ -0,0 +1,28 @@
+from typing import List, Tuple, Union, Callable, Dict, Optional
+
+from lark import Tree
+from lark.grammar import RuleOptions
+
+
+class Grammar:
+    rule_defs: List[Tuple[str, Tuple[str, ...], Tree, RuleOptions]]
```

Comment:
If it's going to be an official interface, it might worth it to create classes for `RuleDef` and `TokenDef`.

It might also make `load_grammar` a bit cleaner.

Reply:
(official interface in the sense that it might be interesting to look at these attributes. Modifying them directly is probably still to be discouraged)

I agree, but we once again run into naming issues since `TerminalDef` is already an existing class. And I would probably just make them named tuples, so that the names are readable.

Reply:
Well, there are a few ways to resolve the naming situation. Like prefix it with `Internal` or rename `TerminalDef`. Just ask yourself, what is the main difference between `lexer.TerminalDef` to the one in load_grammar.